api/src/Entity/Configuration.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Entity;
+
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\Delete;
+use ApiPlatform\Metadata\Get;
+use ApiPlatform\Metadata\GetCollection;
+use ApiPlatform\Metadata\Patch;
+use ApiPlatform\Metadata\Post;
+use App\Repository\ConfigurationRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation\Groups;
+
+#[ORM\Entity(repositoryClass: ConfigurationRepository::class)]
+#[ApiResource(
+    operations: [
+        new Get(),
+        new Post(
+            denormalizationContext: ['groups' => 'create_update_domain_denormalization'],
+        ),
+        new Patch(
+            denormalizationContext: ['groups' => 'create_update_domain_denormalization'],
+        ),
+        new Delete(),
+    ],
+    normalizationContext: ['groups' => 'get_configuration_normalization'],
+    security: "is_granted('ROLE_ADMIN') or object.getDomain().getOwner() == user",
+)]
+class Configuration
```

Comment:
All your properties in your entities are missing the Assert not blank on field that should not be empty.

Reply:
Ensure returned value are not nullable in your getters/setters too.